aws/src/main/python/cloudwatchAllInstances/lambda_function.py

```diff
@@ -113,29 +121,30 @@ def convert_to_ec2_instance_from(response_instance, region):
     logger.info('Convert to EC2Instance from ' + str(response_instance))
     email = None
     tags = response_instance.get('Tags')
-    if tags is not None:
+    if tags != None:
```

Comment:
would it be better to write as 'if tags:'? 
to check also empty string and an empty list.

Reply:
Sure - although it would not make a difference in this case afaict